client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -50,6 +55,18 @@ const FileInfoCard: FunctionComponent< FileInfoCardProps > = ( {
 
 	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) ) as string;
 
+	const doesRewindNeedCredentials = useSelector( ( state ) =>
+		getDoesRewindNeedCredentials( state, siteId )
+	);
+	const isRestoreInProgress = useSelector( ( state ) => getIsRestoreInProgress( state, siteId ) );
+	const isAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, siteId ) );
+	const areCredentialsInvalid = useSelector( ( state ) =>
+		areJetpackCredentialsInvalid( state, siteId, 'main' )
+	);
+
+	const isRestoreDisabled =
```

Comment:
This logic is being used in multiple places right now. Maybe we could group all these selectors into one.